src/wmtk/io/MshReader.hpp

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#include <Eigen/Core>
+#include "MeshReader.hpp"
+
+#include <mshio/mshio.h>
+
+namespace wmtk {
+
+class MshReader : public MeshReader
+{
+protected:
+    void read_aux(const std::filesystem::path& filename, Mesh& mesh) override;
```

Comment:
i think you're still in the middle of writing things but it'd be good to have comments on what aux/cb are in this class